src/wp-includes/class-wp-theme-json-data.php

```diff
@@ -69,4 +69,15 @@ public function update_with( $new_data ) {
 	public function get_data() {
 		return $this->theme_json->get_raw_data();
 	}
+
+	/**
+	 * Returns theme JSON object.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @return WP_Theme_JSON
```

Comment:
As per [WordPress Docblock standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods), we need a line describing what is returned here.

Maybe something like `@return A structure that follows the theme.json schema.` ?

Reply:
Handled as part of the commit.